src/vis/scatter/ScatterVis.tsx

```diff
@@ -136,12 +131,16 @@ export function ScatterVis({
   }, [traces, config.dragMode]);
 
   return (
-    <div ref={plotlyDivRef} className="d-flex flex-row w-100 h-100" style={{ minHeight: '0px' }}>
-      <div
-        className={`position-relative d-flex justify-content-center align-items-center flex-grow-1 ${
-          traceStatus === 'pending' ? 'tdp-busy-partial-overlay' : ''
-        }`}
-      >
+    <Container fluid sx={{ flexGrow: 1, height: '100%', overflow: 'hidden' }} ref={plotlyDivRef}>
+      <ActionIcon sx={{ position: 'absolute', top: '10px', right: '10px' }} onClick={() => setSidebarOpen(true)}>
+        <FontAwesomeIcon icon={faGear} />
+      </ActionIcon>
```

Comment:
Please add a Mantine tooltip:

> Switch visualization type and configure currently selected visualization

Apply this change to all other action icons of the visualization types

Reply:
done